packages/safe-tools-subgraph/networks.json

```diff
@@ -0,0 +1,12 @@
+{
+  "goerli": {
+    "ScheduledPaymentModule": {
+      "startBlock": 7870552
```

Comment:
This is when https://github.com/cardstack/cardstack-module-scheduled-payment/commit/7d285c2bebdd942e94cecc5e1fee4dbce89c9029 was deployed